Jenkinsfile

```diff
@@ -33,6 +33,7 @@ pipeline {
               "-Dgpg.project-build.password='${env.GPG_PWD}' " +
               "-Dgithub.site.skip=${params.skipGitHubSite} " +
               "-Divy.engine.list.url=${params.engineListUrl} " +
+              "-Divy.engine.version='[11.0.0,11.1.2]' " +

```

Comment:
let's shortly discuss whether this is a case for 'project-build-plugin' raise pipeline. actually I don't think so ... but maybe I don't yet understand it fully.